libraries/ms-helper/inc/calib.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+// Calibration persistence - uses the flash persist layer
+// Any changes must be explicitly committed.
+// There can only be one instance of this module.
+#include "persist.h"
+
+// Use the last flash page for calibration persistence
+#define CALIB_FLASH_PAGE NUM_FLASH_PAGES - 1
+
+// Load any stored data into |blob|. |blob| should persist.
+StatusCode calib_init(void *blob, size_t blob_size);
+
+// Store any changes to the calib blob.
```

Comment:
Nit: `s/calib/calibration/` in comments. I'm not a fan of the abbreviation it sounds weird and isn't common but that's more of an opinion.